### PR TITLE
Us110 display trading as

### PR DESCRIPTION
**Pull request by rhiGriff**
Displaying trading as only if there is a value.

### CODE REVIEW

**Comment by rhiGriff** [comment]
[US110: EXT Display "trading as" in To do and History [M]](https://trello.com/c/pXNYDmoS/204-us110-ext-display-trading-as-in-to-do-and-history-m)

File: tests/app/test_surveys.py [hunk 1]
```diff
@@ -70,6 +73,42 @@ def test_surveys_todo(self, mock_request):
         self.assertTrue('Business Register and Employment Survey'.encode() in response.data)
         self.assertTrue('RUNAME1_COMPANY4 RUNNAME2_COMPANY4'.encode() in response.data)
 
+    @requests_mock.mock()
```
Comment (JamesGardiner) [inline tests/app/test_surveys.py]: What happens if the '/surveys/todo` endpoint returns an error?

Comment (rhiGriff) [reply tests/app/test_surveys.py]: This should be covered in another test as these tests are only checking for the trading as and not if the page can be displayed.

File: frontstage/templates/surveys/surveys.html [hunk 2]
```diff
@@ -31,9 +31,12 @@
...
             <span id="SURVEY_NAME">{{ survey.survey.longName }}</span><br/>
-            <span class="survey-list-item pluto">
-                        BUSINESS: {{ survey.business_party.name }}<br/>
-                        REF: {{survey.case.caseGroup.sampleUnitRef }}
+            <span id="REPORTING_UNIT_DETAILS_{{ survey.case.caseGroup.sampleUnitRef }}" class="survey-list-item pluto">
+                        Business: {{ survey.business_party.name }}<br/>
+                        {% if survey.business_party.trading_as %}
+                        Trading as: {{survey.business_party.trading_as }}<br/>
+                        {% endif %}
+                        Ref: {{survey.case.caseGroup.sampleUnitRef }}
```
Comment (emilioward) [inline frontstage/templates/surveys/surveys.html]: Can we fix the indentation spacing on this please

**Pull request merged**